defender/downloaded/Forta_Governance.js

```diff
@@ -238,56 +244,51 @@ async function createDiscordMessage(metadata, description, alertId, transactionH
 exports.handler = async function (autotaskEvent) {
   // ensure that the autotaskEvent Object exists
   if (autotaskEvent === undefined) {
-    return {};
+    throw new Error('autotaskEvent undefined');
   }
-  console.log('autotaskEvent');
-  console.log(JSON.stringify(autotaskEvent, null, 2));
 
-  const { secrets } = autotaskEvent;
+  const { secrets, request } = autotaskEvent;
   if (secrets === undefined) {
-    return {};
+    throw new Error('secrets undefined');
   }
 
   // ensure that there is a DiscordUrl secret
-  const { TestingDiscordUrl: discordUrl } = secrets;
+  const { GovernanceDiscordUrl: discordUrl } = secrets;
   if (discordUrl === undefined) {
-    return {};
+    throw new Error('GovernanceDiscordUrl undefined');
   }
 
   // ensure that the request key exists within the autotaskEvent Object
-  const { request } = autotaskEvent;
   if (request === undefined) {
-    return {};
+    throw new Error('request undefined');
   }
 
   // ensure that the body key exists within the request Object
   const { body } = request;
   if (body === undefined) {
-    return {};
+    throw new Error('body undefined');
   }
 
   // ensure that the alert key exists within the body Object
   const { alert } = body;
   if (alert === undefined) {
-    return {};
+    throw new Error('alert undefined');
   }
 
-  // extract the transaction hash and agent ID from the alert Object
+  // extract the transaction hash and bot ID from the alert Object
   const {
     hash,
     source: {
       transactionHash,
       agent: {
```

Comment:
should this be:
```suggestion
      bot: {
```
or is this one still `agent`?

Reply:
I'm not certain, as this is the alert that comes from the Forta Sentinel.  It may depend upon how they are crafting the Object that is sent to the Autotask (or it's just from JSON).

Reply:
It looks like both work, but we should use the latest.  I'll change them to `bot` in this location.